### ISSUE TITLE
Can I use the models stored on the network drive with a pod running the Comfy GUI?

### ISSUE THREAD

**Issue by okkazoo** [blib-la/runpod-worker-comfy#72]
Thank you for this. I have the headless version of flux-dev running well.

I'm a little new to runpod so apologies for my ignorance. I want to use a pod with the comfy GUI to create specific workflows. I'm a little confused as to whether I need to have duplicate copies of models and custom nodes for this? 

I have copied some checkpoints to here /runpod-volume/models/checkpoints/ for use with the serverless worker but I can't seem to access these within the Comfy GUI? The models it looks for are obviously in /workspace/ComfyUI/models/checkpoints/

I am running the comfy GUI on a runpod with the template "Runpod SD ComfyUI". I can see /models/checkpoints/ but I'm not sure if this is even possible?

**Comment by okkazoo** [comment]
Actually answered my own question. Created a symbolic link to the  /runpod-volume/models/checkpoints/

**Comment by TimPietruskyRunPod** [comment]
Yes — a pod (not serverless) can mount the same network volume that your serverless endpoint uses, with the difference that on a pod the volume mounts at \`/workspace\` (not \`/runpod-volume\`). So if you have a model at the network-volume path \`models/checkpoints/myckpt.safetensors\`, the pod sees it at \`/workspace/models/checkpoints/myckpt.safetensors\`. That's covered in https://github.com/runpod-workers/worker-comfyui/blob/main/docs/network-volumes.md and clarified further in PR #154 (open) and PR #193 (open).

To run the **ComfyUI GUI** on a pod against the same models the serverless worker uses:

\`\`\`bash
runpodctl pod create --image runpod/worker-comfyui:5.8.5-base --network-volume-id <your-volume-id> --gpu-id "NVIDIA GeForce RTX 4090"
\`\`\`

The GUI is reachable on port 8188 via the proxy URL.

Closing as answered.
